contrib/tester-progs/go/lseek-pipe/lseek-pipe.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"strconv"
+
+	"golang.org/x/sys/unix"
+)
+
+// This program is used to test tracing policies. It can be called in two ways:
+//  - with three arguments: it uses the arguments to execute an lseek syscall
+//  - with no arguments, it three arguments from stdin, and executes itself with them
```

Comment:
typo `... arguments, it three arguments ...` -> `... arguments: it uses three arguments ...`

Reply:
thanks, fixed!